src/main/scala/dotty/dokka/tasty/PackageSupport.scala

```diff
@@ -0,0 +1,46 @@
+package dotty.dokka.tasty
+
+import org.jetbrains.dokka.model._
+import org.jetbrains.dokka.links._
+import org.jetbrains.dokka.model.properties._
+import org.jetbrains.dokka.model.doc.DocumentationNode
+
+import collection.JavaConverters._
+
+trait PackageSupport:
+    self: TastyParser =>
+    import reflect._
+    import self.SymbolOps.{documentation}
+    def parsePackage(pck: PackageClause): DPackage = {
+        val name = extractPackageName(pck.pid.show)
+        val documentation = pck.symbol.documentation
```

Comment:
@AleksanderBG do you recall any case where `pck.symbol.documentation` can return anything usefull?

Reply:
No, I don't think it ever does.